hack/tools/catalogs/list-compatible-bundles

```diff
@@ -77,7 +77,7 @@ function extract-olm-package-property() {
 
 # Group packages by name and collect versions into an array
 function group-versions-by-package-name() {
-    jq -s 'group_by(.packageName) | map({packageName: .[0].packageName, versions: map(.version)})' | regex_it
+    jq -s 'group_by(.packageName) | map({packageName: .[0].packageName, versions: map(.version)})'
```

Comment:
WDYT about in the description of this change  (assuming that we have a PR to address this specific need), you add the result before the change and after so that we can easily follow up on what you are trying to do/fix here?

Reply:
let me add it to current description to potentially re-use in case of separating PR's